src/avrodoc.js

```diff
@@ -62,7 +62,7 @@ function readJSON(filename) {
     try {
         parsed = JSON.parse(json);
     } catch (e) {
-        sys.error('Not a valid json file: ' + filename);
+        console.error('Not a valid json file: ' + filename);
```

Comment:
In this file `sys` isn't even imported, so this is failing on `sys is not defined`.